src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = props => {
+  let dateOfBirthFormatted = '';
+  let genderFull = '';
+  if (props.user && props.user.dob) {
+    dateOfBirthFormatted = moment(props.user.dob).format('MMMM Do YYYY');
+  }
+  if (props.user && props.user.gender === 'M') {
+    genderFull = 'Male';
+  } else if (props.user && props.user.gender === 'F') {
+    genderFull = 'Female';
+  }
+  const alertContent = (
+    <>
+      <p>
+        <strong>
+          {props.user.userFullName.first} {props.user.userFullName.last}
+        </strong>
+      </p>
+      <p>Date of birth: {dateOfBirthFormatted}</p>
+      <p>Gender: {genderFull}</p>
+    </>
+  );
+  return (
+    <div>
+      <p>This is the information we have on file for you.</p>
+      <AlertBox content={alertContent} status="info" isVisible />
```

Comment:
This can be used, but needs to align with [our mockups](https://xd.adobe.com/view/ca825c11-2268-458f-4fec-6aaecfec5b2d-bcb3/screen/dc153448-40b1-4d35-b890-698dca690305/Add-Modify-Dependents-MVI-Details) which call for a white background, no icon, and a different color for the left border. 

You can pass `className="vads-u-background-color--white vads-u-border-color--primary usa-alert-info::before"` to AlertBox, and it should work properly. `.usa-alert-info::before` will need to be added as custom css to the scss file for the 686 since it needs to override the default class. Top and bottom padding might also need to be changed to align properly with our mockups.

Reply:
Nice! I had no idea these classes existed